samples/python/hist.py

```diff
@@ -46,9 +46,9 @@ def hist_lines(im):
         im = cv.cvtColor(im,cv.COLOR_BGR2GRAY)
     hist_item = cv.calcHist([im],[0],None,[256],[0,256])
     cv.normalize(hist_item,hist_item,0,255,cv.NORM_MINMAX)
-    hist=np.int32(np.around(hist_item))
+    hist = np.around(hist_item)
     for x,y in enumerate(hist):
-        cv.line(h,(x,0),(x,y[0]),(255,255,255))
+        cv.line(h,(x,0),(x,int(y)),(255,255,255))
```

Comment:
Why we should have different code on 3.4 and master branch?

Reply:
on 3.4 branch there was array of one integer element, on master branch - of one float

Reply:
There was `hist=np.int32(...)` above.

Reply:
yes, I missed it, thanks